src/__tests__/components/modal.test.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import Modal from 'react-modal';
+import configureStore from 'redux-mock-store';
+import { mount } from 'enzyme';
+import { MemoryRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import thunk from 'redux-thunk';
+import OptionModal from '../../components/modal';
+
+const props = {
+  article: {},
+  closeModal: jest.fn(),
+  deleteArticle: jest.fn(),
+  displayModal: jest.bool,
+  history: {
+    push: jest.fn(),
+    location: {
+      pathname: ''
+    }
+  },
+};
+
+const article = {
+  id: 1,
+  title: 'test',
+  description: 'description',
+  images: ['image'],
+  body: 'body',
+  updatedAt: 'time',
+  slug: 'slug',
+};
+
+const mockStore = configureStore([thunk]);
+
+const store = mockStore({
+  articleReducer: {
+    article,
+  }
+});
+
+const mountWrapper = mount(
+  <Provider store={store}>
+    <MemoryRouter>
+      <OptionModal {...props} />
+    </MemoryRouter>
+  </Provider>
+);
+
+describe('Modal', () => {
```

Comment:
@chingsley is this a component you created yourself? If it is not, you shouldn't be testing components exposed by a library.

Reply:
Aright. I will amend the test exclude such components.

Reply:
Done